src/uct/cuda/cuda_ipc/cuda_ipc_ep.c

```diff
@@ -15,6 +15,12 @@
 #define UCT_CUDA_IPC_PUT 0
 #define UCT_CUDA_IPC_GET 1
 
+SGLIB_DEFINE_LIST_FUNCTIONS(uct_cuda_ipc_rem_seg_t,
```

Comment:
sglib is going to be deprecated, newer code should use khash

Reply:
@yosefe  @brminich I'll move to this but I see that even mm uct still uses sglib.

Reply:
you are right, we have some leftovers which use sglib. we prefer all new code to use khash since it grows automatically